warehouse/manage/views.py

```diff
@@ -3018,6 +3029,13 @@ def transfer_organization_project(project, request):
             request.route_path("manage.project.settings", project_name=project.name)
         )
 
+    # Check that user has permission to remove projects from organization.
+    if project.organization and request.user not in project.organization.owners:
+        request.session.flash("Could not transfer project", queue="error")
```

Comment:
Should we add a reason to this message? Perhaps "Could not transfer project, you do not have the required permissions". Otherwise people may interpret it as a system error.

Reply:
Added clearer error message in commit [6afafa0](https://github.com/pypi/warehouse/pull/12008/commits/6afafa0479efa813257dafb90dd67a196d35dae1).